Sources/MapboxSearch/PublicAPI/Search Results/SearchSuggestion.swift

```diff
@@ -44,5 +44,7 @@ public protocol SearchSuggestion {
     /// Indicates whatever this suggestion can be batch resolved. Suggestion should be POI type.
     var batchResolveSupported: Bool { get }
 
+    /// An estimated time of arrival (in minutes) based on requested proximity  May be `nil` even for correct proximity
+    /// argument.
```

Comment:
```suggestion
    /// An estimated time of arrival (in minutes) based on requested proximity.
    /// It can be `nil` even for the correct proximity argument.
```